### ISSUE TITLE
finetuning the model on high resolution novel view synthesis

### ISSUE THREAD

**Issue by VillardX** [QitaoZhao/E-RayZer#13]
Hi @QitaoZhao,

I finetuned the erayzer-multi model on my own dataset.

When using the default resolution (256), the results look good.  
However, when increasing the resolution to 336 / 448 / 896, I observe:

- noticeable loss of fine texture details in novel views  
- boundary sticking artifacts in the reconstructed point cloud  

I suspect this might be related to the model capacity or the current image tokenizer / decoder design not scaling well to higher resolutions.

I’m wondering:

1. Have you explored training erayzer at higher resolutions?  
   If so, what were the main challenges?

2. Do you think replacing components like:
   - patch embedding (e.g., using DINOv2-style patch_embed)
   - decoder (e.g., DPT-style decoder)  
   could help improve high-resolution performance?

3. Compared to methods like VGGT (~1B params, supervised),  
   erayzer is relatively lightweight (<400M params) and self-supervised.  
   Do you think erayzer has the potential to scale (in terms of model size or training strategy) to better handle high-resolution reconstruction?

Thanks!

**Comment by QitaoZhao** [comment]
Hi, thanks for the great question!

We have not yet had the chance to train our model on high-resolution images, so I cannot make a definitive claim based on experiments. That said, I do not see any principled reason why the method would not work at higher resolutions. My guess is that training from scratch, rather than fine-tuning, would likely be the more reliable way to make it work well. I also personally do not think the current model architecture itself would be a major bottleneck for high-resolution performance (some architecture tweaks, as you mentioned, may make it work better though).

We are still working on releasing the training code.

**Comment by VillardX** [comment]
Thanks for your suggestions~
